### PR TITLE
Changelog release-3.0

### PR DESCRIPTION
**Pull request by shiyuhang0**
### What problem does this PR solve? 
- add changelog
- revision

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -92,7 +92,7 @@
         <scalaj.version>2.3.0</scalaj.version>
         <lombok.version>1.18.22</lombok.version>
         <someModule.test.excludes></someModule.test.excludes>
-        <revision>3.0.0</revision>
+        <revision>3.0.1</revision>
```
Comment (xuanyu66) [inline pom.xml:95]: Snapshot？

**Review by zhangyangyu — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+- Support DELETE statement [#2276](https://github.com/pingcap/tispark/pull/2276)
+- Support Spark 3.2 [#2287](https://github.com/pingcap/tispark/pull/2287)
+- Support telemetry to collect information [#2316](https://github.com/pingcap/tispark/pull/2316)
+- Support stale read to read historical versions of data [#2322](https://github.com/pingcap/tispark/pull/2322)
+- Support TLS with reload capability [#2306](https://github.com/pingcap/tispark/pull/2306) [#2349](https://github.com/pingcap/tispark/pull/2349) [#2365](https://github.com/pingcap/tispark/pull/2349) [#2377](https://github.com/pingcap/tispark/pull/2377)
+
+### Fixes
+- Fix the wrong result of _tidb_rowid when set `spark.tispark.show_rowid=true` [#2270](https://github.com/pingcap/tispark/pull/2270)
+- Fix sum not push down bug [#2314](https://github.com/pingcap/tispark/pull/2314)
+- Fix limit not push down bug [#2329](https://github.com/pingcap/tispark/pull/2329)
```
Comment (Daemonxiao) [inline CHANGELOG.md:19]: Is there any syntax error here? How about "Fix sum can't be pushed down bug"?

Comment (xuanyu66) [reply CHANGELOG.md:19]: Fix the bug that the sum filter can't be pushed down

**Review by Daemonxiao — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (shiyuhang0) [reply CHANGELOG.md:19]: For consistency with releases and release emails. I suggest not change it now.

**Review by xuanyu66 — APPROVED** [review]

**Pull request merged**